python/cudf/cudf/tests/test_parquet.py

```diff
@@ -3614,14 +3614,15 @@ def test_parquet_writer_roundtrip_with_arrow_schema(index):
         }
     )
 
+    # Convert decimals32/64 to decimal128 if pyarrow version is < 19.0.0
```

Comment:
Instead of skipping the test entirely for pyarrow < 19, this change sync's this test with #17869.